### PR TITLE
fix(http): links to Keep-Alive specs

### PR DESCRIPTION
**Pull request by bsmth**
This points to a working link for Keep-Alive.

Note that the [Keep-Alive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive) page points to https://www.rfc-editor.org/rfc/rfc9112#compatibility.with.http.1.0.persistent.connections (rfc9112). But in this table, the [spec link](https://datatracker.ietf.org/doc/html/draft-thomson-hybi-http-timeout-03#section-2) is marked as experimental and expired. 

Should we also link to https://www.rfc-editor.org/rfc/rfc9112#compatibility.with.http.1.0.persistent.connections with a note or is this edit sufficient?

### CODE REVIEW

File: files/en-us/web/http/resources_and_specifications/index.md [hunk 1]
```diff
@@ -8,49 +8,49 @@ page-type: guide
...
+| {{rfc(3986)}}                                                                                           | Uniform Resource Identifier (URI): Generic Syntax                                                                                                                                                                                                     | Internet Standard        |
+| {{rfc(5988)}}                                                                                           | Web Linking _Defines the {{HTTPHeader("Link")}} header_                                                                                                                                                                                               | Proposed Standard        |
+| [Experimental spec](https://datatracker.ietf.org/doc/html/draft-thomson-hybi-http-timeout-03#section-2) | Hypertext Transfer Protocol (HTTP) Keep-Alive Header                                                                                                                                                                                                  | Informational (Expired)  |
```
Comment (teoli2003) [inline files/en-us/web/http/resources_and_specifications/index.md:30]:
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Given that this proposal has been abandoned, we should:
- remove the entry in the list here (no need to fix it, it is irrelevant)
- keep the spec link in the table (to HTTP/1.1) in the Keep-Alive page
- remove the link to the abandoned experimental in the Keep-Alive page.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**